### PR TITLE
[PL-46816] Update Single Sign-On (SSO) with SAML SCIM API settings details

### PR DESCRIPTION
**Pull request by brian-f-harness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: Update Single Sign-On (SSO) with SAML with SCIM API settings details
* Jira/GitHub Issue numbers (if any): PL-46816
* Preview links/images (Internal contributors only): [SCIM API integration settings](https://663001f04ee4ca23731389c3--harness-developer.netlify.app/docs/platform/authentication/single-sign-on-saml#scim-api-integration-settings)

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] Code owner review.
- [x] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/platform/authentication/single-sign-on-saml.md [hunk 1]
```diff
@@ -14,18 +14,40 @@ Harness supports Single Sign-On (SSO) with SAML, integrating with your SAML SSO
...
 
+## Use System for Cross-domain Identity Management (SCIM) protocol
+
+To ensure continuous and real-time synchronization of user group bindings and access controls, it is recommended to utilize the System for Cross-domain Identity Management (SCIM) protocol. SCIM enables real-time syncing of user additions with Harness user groups, ensuring that user permissions and access rights are consistently applied and maintained.
+
+For implementation details and further guidance on provisioning users with SCIM, go to [Okta SCIM](/docs/platform/role-based-access-control/provision-users-with-okta-scim), [Microsoft Entra SCIM](/docs/platform/role-based-access-control/provision-users-and-groups-using-azure-ad-scim), and [OneLogin SCIM](/docs/platform/role-based-access-control/provision-users-and-groups-with-one-login-scim).
+
+### SCIM API integration settings
+
+If you provision users and groups via SCIM API, use the following for your SAML integration.
```
Comment (brian-f-harness) [inline docs/platform/authentication/single-sign-on-saml.md:36]:
```suggestion
If you provision users and groups via SCIM API, use the following settings for your SAML integration.
```

File: docs/platform/authentication/single-sign-on-saml.md [hunk 2]
```diff
@@ -14,18 +14,40 @@ Harness supports Single Sign-On (SSO) with SAML, integrating with your SAML SSO
...
+For implementation details and further guidance on provisioning users with SCIM, go to [Okta SCIM](/docs/platform/role-based-access-control/provision-users-with-okta-scim), [Microsoft Entra SCIM](/docs/platform/role-based-access-control/provision-users-and-groups-using-azure-ad-scim), and [OneLogin SCIM](/docs/platform/role-based-access-control/provision-users-and-groups-with-one-login-scim).
+
+### SCIM API integration settings
+
+If you provision users and groups via SCIM API, use the following settings for your SAML integration.
+
+- **SCIM connector base URL:** `https://app.harness.io/gateway/ng/api/scim/account/<YOUR_ACCOUNT_ID`
+   - Replace `YOUR_ACCOUNT_ID` with your Harness account ID.
+   The URL depends on the Harness production cluster you use: Prod1: `https://app.harness.io`, Prod2: `https://app.harness.io/gratis`, or Prod3: `https://app3.harness.io`.
+- **Unique identifier:** field for users: `userName`
```
Comment (brian-f-harness) [inline docs/platform/authentication/single-sign-on-saml.md:41]:
```suggestion
- **Unique identifier:** `userName`
```

File: docs/platform/authentication/single-sign-on-saml.md [hunk 3]
```diff
@@ -14,18 +14,40 @@ Harness supports Single Sign-On (SSO) with SAML, integrating with your SAML SSO
...
+- **Authentication Mode:** HTTP Header
+- **Authorization:** `<YOUR_SERVICE_ACCOUNT_TOKEN>`
+
+You must also do the following:
+
+- Enable provisioning to Harness.
+- Assign your user groups.
+- Push your groups.
```
Comment (brian-f-harness) [inline docs/platform/authentication/single-sign-on-saml.md:49]:
```suggestion
- Push your groups to Harness.
```

**Review by piyush-harness — APPROVED** [review]

File: docs/platform/authentication/single-sign-on-saml.md [hunk 4]
```diff
@@ -14,18 +14,40 @@ Harness supports Single Sign-On (SSO) with SAML, integrating with your SAML SSO
...
 
-To ensure continuous and real-time synchronization of user group bindings and access controls, it is recommended to utilize the System for Cross-domain Identity Management (SCIM) protocol. SCIM enables real-time syncing of user additions with Harness user groups, ensuring that user permissions and access rights are consistently applied and maintained. For implementation details and further guidance on provisioning users with SCIM, go to [Okta SCIM](/docs/platform/role-based-access-control/provision-users-with-okta-scim), [Microsoft Entra SCIM](/docs/platform/role-based-access-control/provision-users-and-groups-using-azure-ad-scim), and [OneLogin SCIM](/docs/platform/role-based-access-control/provision-users-and-groups-with-one-login-scim).
 :::
 
+## Use System for Cross-domain Identity Management (SCIM) protocol
+
+To ensure continuous and real-time synchronization of user group bindings and access controls, it is recommended to utilize the System for Cross-domain Identity Management (SCIM) protocol. SCIM enables real-time syncing of user additions with Harness user groups, ensuring that user permissions and access rights are consistently applied and maintained.
```
Comment (douglas-j-bothwell) [inline docs/platform/authentication/single-sign-on-saml.md:30]:
```suggestion
To ensure continuous and real-time synchronization of user group bindings and access controls, Harness recommends that you utilize the System for Cross-domain Identity Management (SCIM) protocol. SCIM enables real-time syncing of user additions with Harness user groups, ensuring that user permissions and access rights are consistently applied and maintained.
```

File: docs/platform/authentication/single-sign-on-saml.md [hunk 5]
```diff
@@ -102,21 +126,21 @@ Sometimes users might have mixed case email addresses in Okta. In these situatio
...
 
-6. In **Audience URI (SP Entity ID)**, enter `app.harness.io`. The SAML application identifier is always `app.harness.io`.
-7. In **Default RelayState**, enter a valid URL. This is the page where users land after a successful sign-in using SAML into the SP.
-8. In **Name ID format**, enter the username format you are sending in the SAML Response. The default format is **Unspecified**.
-9. In **Application username**, enter the default value to use for the username with the application.
-10. In **Attribute Statements (optional)**, enter name in the **Name** field, select **Name Format** as **Basic**, and select the **Value** as **user.email**.
+5. In **Audience URI (SP Entity ID)**, enter `app.harness.io`. The SAML application identifier is always `app.harness.io`.
+6. In **Default RelayState**, enter a valid URL. This is the page where users land after a successful sign-in using SAML into the SP.
+7. In **Name ID format**, enter the username format you are sending in the SAML Response. The default format is **Unspecified**.
+8. In **Application username**, enter the default value to use for the username with the application.
```
Comment (douglas-j-bothwell) [inline docs/platform/authentication/single-sign-on-saml.md:132]:
```suggestion
8. In **Application username**, enter the default username.
```

File: docs/platform/authentication/single-sign-on-saml.md [hunk 6]
```diff
@@ -334,17 +358,17 @@ When a user has many group memberships, the number of groups listed in the token
...
+1. In your Azure, go to **App registrations**.
 2. Click on your app. Copy the Application (client) ID and paste it in **Client ID** in your Harness account.
-3. In your  account, go to **App registrations**. Click **Certificates and Secrets**.
-4. Click New Client Secret.
+3. In your  account, go to **App registrations**. Click **Certificates and Secrets**.
+4. Select **New Client Secret**.
 5. Add a description and click **Add**.
-6. Make sure to copy this secret and save it as an encrypted text secret. For detailed steps to create an encrypted text in Harness, go to [Use Encrypted text Secrets](../secrets/add-use-text-secrets).
-7. Select the above secret reference in the Client Secret field in your Harness account.
+6. Make sure to copy this secret and save it as an encrypted text secret. For detailed steps to create an encrypted text in Harness, go to [Use Encrypted text Secrets](../secrets/add-use-text-secrets).
```
Comment (douglas-j-bothwell) [inline docs/platform/authentication/single-sign-on-saml.md:370]:
```suggestion
6. You must copy this secret and save it as an encrypted text secret. For detailed steps to create an encrypted text in Harness, go to [Use Encrypted text Secrets](../secrets/add-use-text-secrets).
```

File: docs/platform/authentication/single-sign-on-saml.md [hunk 7]
```diff
@@ -555,26 +579,25 @@ For detailed steps on adding SAML SSO with Microsoft Entra ID, follow the steps
...
 
-1. In your Microsoft Entra ID account, go to **App registrations**.
+1. In your Microsoft Entra ID account, go to **App registrations**.
 2. Click on your app. Copy the Application (client) ID and paste it in **Client ID** in your Harness account.
-3. In your Microsoft Entra ID account, go to **App registrations**. Click **Certificates and Secrets**.
+3. In your Microsoft Entra ID account, go to **App registrations**. Click **Certificates and Secrets**.
 4. Click New Client Secret.
 5. Add a description and click Add.
-6. Make sure to copy this secret and save it as an encrypted text secret. For detailed steps to create an encrypted text in Harness, go to [Use Encrypted text Secrets](/docs/platform/secrets/add-use-text-secrets).
+6. Make sure to copy this secret and save it as an encrypted text secret. For detailed steps to create an encrypted text in Harness, go to [Use Encrypted text Secrets](/docs/platform/secrets/add-use-text-secrets).
```
Comment (douglas-j-bothwell) [inline docs/platform/authentication/single-sign-on-saml.md:597]:
```suggestion
6. You must copy this secret and save it as an encrypted text secret. For detailed steps to create an encrypted text in Harness, go to [Use Encrypted text Secrets](/docs/platform/secrets/add-use-text-secrets).
```

File: docs/platform/authentication/single-sign-on-saml.md [hunk 8]
```diff
@@ -587,41 +610,41 @@ Enabling OneLogin authentication on Harness requires configuration on both platf
...
-   
+7. In **Configuration**, paste this URL into the **SCIM Base URL** field.
+
    ![](./static/single-sign-on-saml-102.png)
 
-8. Skip all other **Application Details** fields, and click **Save**.
-9.  Navigate to OneLogin's **Applications** > **SSO** tab. At the upper right, select **More Actions** > **SAML Metadata**.
-    
+8. Skip all other **Application Details** fields, and click **Save**.
+9.  Navigate to OneLogin's **Applications** > **SSO** tab. At the upper right, select **More Actions** > **SAML Metadata**.
```
Comment (douglas-j-bothwell) [inline docs/platform/authentication/single-sign-on-saml.md:632]:
```suggestion
9. Navigate to OneLogin's **Applications** > **SSO** tab. At the upper right, select **More Actions** > **SAML Metadata**.
```

**Review by douglas-j-bothwell — COMMENTED** [review]
A few word-nerd sugs otherwise LGTM

**Pull request merged**